docs/redirectless.md

```diff
@@ -33,7 +33,7 @@ authnWidget.initRedirectless(config);
 ## Configuration object
 There are multiple options for creating the configuration object to initiate the redirectless flow:
 1. Create a configuration object that contains the `onAuthorizationSuccess` function and the required attributes (such as `client_id`, `response_type`, etc.) used by the internal authorization request function.
-1. Create a configuration object containing the `onAuthorizationSuccess` function with the `interactWith` attribute set to `PfAuthnWidget.INTERACT_WITH_USER_AUTHZ`. This configuration initiates the Authentication API Widget to interact with the OAuth 2.0 Device Authorization's User Authorization endpoint.
+1. Create a configuration object containing the `onAuthorizationSuccess` function with the `flowType` attribute set to `PfAuthnWidget.FLOW_TYPE_USER_AUTHZ`. This configuration initiates the Authentication API Widget to interact with the OAuth 2.0 Device Authorization's User Authorization endpoint. Optionally, `user_code` attribute can be provided and passed to the user authorization endpoint as a query parameter. You can find more information about the Authentication API widget OAuth 2.0 Device Authorization configuration [here](#oauth-2.0-device-authorization).
```

Comment:
Suggested:

> <...> This configuration initializes the Authentication API Widget to interact with PingFederate's user authorization endpoint. Optionally, the `user_code` attribute can be provided. If provided, it is passed to the user authorization endpoint as a query parameter, which will trigger the a state where the user must confirm the code (rather than having to enter it). <...>

Also -- where does #oauth-2.0-device-authorization link to?